README.md

```
@@ -1,105 +1,134 @@
 
-Contributions are welcome, please open an issue or a pull request.
+PriceBuddy is built with [Laravel](https://laravel.com) and
+[Filament](https://filamentphp.com/), with a [Lando](https://lando.dev)-based dev
+environment (`lando start`). Coding standards (Pint), static analysis (PHPStan)
+and tests (Pest/PHPUnit) are enforced. See the
+[development docs](https://pricebuddy.jez.me/advanced.html) for the full setup.
 
 ## Supporting development
 
-Have a look [here](https://pricebuddy.jez.me//support-project.html) for ways to 
+Have a look [here](https://pricebuddy.jez.me/support-project.html) for ways to
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)